extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -598,13 +643,14 @@ export class SafeframeHostApi {
       // to execute upon the next user interaction. We don't want
       // that for safeframe, so we reset it here.
       this.baseInstance_.getResource().resetPendingChangeSize();
-      if (optIsCollapse) {
-        // If this is a collapse request, then even if resizing
+      if (isReducing) {
```

Comment:
No need to declare isReducing in advance, can just put logic here

Reply:
done